src/frontend/components/SøknadsSteg/OmDeg/useOmdeg.tsx

```diff
@@ -115,9 +118,33 @@ export const useOmdeg = (): {
         søker.værtINorgeITolvMåneder,
         'personopplysninger.feilmelding.janei',
         { borPåRegistrertAdresse: { jaNeiSpm: borPåRegistrertAdresse } },
-        true
+        borPåRegistrertAdresse.verdi === ESvar.NEI
     );
 
+    const komTilNorgeDato = useDatovelgerFelt(
+        søker.komTilNorgeDato,
+        'omdeg.spm.dato.feil',
+        ESvar.NEI,
+        værtINorgeITolvMåneder
+    );
+
+    const planleggerÅBoINorgeTolvMnd = useFelt<ESvar | undefined>({
+        feltId: søker.planleggerÅBoINorgeTolvMnd.id,
```

Comment:
Trengte veldig lite logikk til den sammenlignet med useJaNeiSpmFelt, så den er lagt til som en egen useFelt istedet for å bruke useJaNeiSpmFelt